implementations/rust/ockam/ockam_command/src/kafka/inlet/create.rs

```diff
@@ -0,0 +1,191 @@
+use async_trait::async_trait;
+use std::net::SocketAddr;
+use std::sync::Arc;
+
+use clap::{command, Args};
+use colorful::Colorful;
+use miette::miette;
+use ockam_api::nodes::models::services::{
+    StartKafkaDirectRequest, StartKafkaRequest, StartServiceRequest,
+};
+use ockam_api::nodes::BackgroundNodeClient;
+use tokio::sync::Mutex;
+use tokio::try_join;
+
+use ockam_api::port_range::PortRange;
+use ockam_core::api::Request;
+use ockam_multiaddr::MultiAddr;
+use ockam_node::Context;
+
+use crate::kafka::util::make_brokers_port_range;
+use crate::node::util::initialize_default_node;
+use crate::service::start::start_service_impl;
+use crate::util::process_nodes_multiaddr;
+use crate::{
+    fmt_log, fmt_ok,
+    kafka::{kafka_default_consumer_server, kafka_inlet_default_addr},
+    node::NodeOpts,
+    util::parsers::socket_addr_parser,
+    Command, CommandGlobalOpts, OckamColor,
+};
+
+/// Create a new Kafka Inlets. Kafka clients v3.7.0 and earlier are supported. You can find the version you have with 'kafka-topics.sh --version'.
+#[derive(Clone, Debug, Args)]
+pub struct CreateCommand {
+    #[command(flatten)]
+    pub node_opts: NodeOpts,
+    /// The local address of the service
+    #[arg(long, default_value_t = kafka_inlet_default_addr())]
+    pub addr: String,
+    /// The address where to bind and where the client will connect to alongside its port, <address>:<port>.
+    /// In case just a port is specified, the default loopback address (127.0.0.1:4000) will be used
+    #[arg(long, default_value_t = kafka_default_consumer_server(), value_parser = socket_addr_parser)]
+    pub from: SocketAddr,
+    /// The address of the kafka bootstrap broker
+    #[arg(long)]
+    pub bootstrap_server: Option<String>,
+    /// Local port range dynamically allocated to kafka brokers, must not overlap with the
+    /// bootstrap port
+    #[arg(long)]
+    pub brokers_port_range: Option<PortRange>,
+    /// The route to the Kafka outlet node, either the project in ockam orchestrator or a rust node, expected something like /project/<name>
+    #[arg(long)]
+    pub to: Option<MultiAddr>,
```

Comment:
If only `bootstrap_server` or `to` can be used at the same time, you could add `conflicts_with = "bootstrap_server"` here.